_pages/en/strategies-actions.md

```diff
@@ -40,49 +44,65 @@ Format:
 #### Cultural
 
 - **IT teams must use at least 10% of their time to learn/try new technologies, attend events/conferences, and stay up to date on trends and technologies related to their work**
-  - *Make it a requirement in performance agreements of managers and team leads, or mandatory for all IT teams*
-  - *Communications about the updated rules around learning and regular updates on upcoming learning opportunities*
-  - *Organize biweekly "Tech Talks" to share knowledge within IITB (maybe this is IITB Showcase)*
+  - *Make it a requirement in performance agreements of directors, managers and team leads*
+  - *Track using existing CATS code for learning (300)*
+  - *Communications about the updated direction around learning and regular updates on upcoming learning opportunities*
+  - *Leverage IITB showcase and Dev CoP to share knowledge within IITB and show progress*
+  - *Create champions for topics (Agile, Git, Development, CI/CD, EA, Testing, Security, ..)*
   - *Create crowdsourced list of learning resources in GCpedia (link on Intranet)*
   - *Create digital channel for sharing (e.g. Slack)*
   - *Encourage Bi-Daily Team Standups to share learning out loud*
+- **Create a safe environment for experimentation/failure**
```

Comment:
Who's responsible for this item? It isn't really at the team level, so it's at the branch level? Or is the Cloud CoE, Sade, etc.?

Reply:
Also not really an action ... what is the action to make this a thing? For example, yesterday we spoke about the creation for a self-service virtual lab with no restrictions (though cannot be connected back into the network). That's an action (given that this document is "strategies and actions")?

Reply:
"tangible" actions are below that line
- *Communications about failures and lessons learned - link back to learning opportunities*
- *Offer cloud and desktop sandboxes for experimentation*
- *Hold blameless postmortems*
- *Organize lunch between the DM and a team/project that recently "failed"*

Reply:
OK - Resolved then.